README.md

```diff
@@ -214,7 +235,7 @@ export const dashboardId = 1
 export const lookId = 1
 ```
 
-* Edit the `demo/demo_user.json` file to be appropriate for the type of user you want to embed.
+* Edit the `demo/demo_user.json` file to be appropriate for the type of user you want to embed. Normally your backend service would use information about the user logged into your embedding application (e.g your customer portal) to inform Looker about important user properties that control data access grants.
```

Comment:
"access grants" has a loaded meaning in the Looker context. I'd suggest "access controls" instead